### DISCUSSION TITLE
CMAKE_MODULE_PATH this project

### DISCUSSION THREAD

**Discussion by autoprog** [aws/aws-iot-device-sdk-cpp-v2#663] [Q&A] [answered]
hi,
new install cpp-v2 error.

example: https://github.com/aws/aws-iot-device-sdk-cpp-v2/issues/372

1. Build C++ AWS IoT SDK dependencies within short path (e.g. C:\dev\iotsdk)
mkdir sdk-cpp-workspace
cd sdk-cpp-workspace
git clone --recursive https://github.com/aws/aws-iot-device-sdk-cpp-v2.git
mkdir aws-iot-device-sdk-cpp-v2-build
cd aws-iot-device-sdk-cpp-v2-build
cmake -DCMAKE_INSTALL_PREFIX="C:\dev\iotsdk\sdk-cpp-workspace" ../aws-iot-device-sdk-cpp-v2
cmake --build . --target install --config "Release"
....ok
see:
![image](https://github.com/aws/aws-iot-device-sdk-cpp-v2/assets/5191545/980827bf-14b3-4491-86a7-a35ac7c590d6)

problems:
By not providing "Findaws-crt-cpp.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "aws-crt-cpp", but CMake did not find one.

I tried several ways to add the module and it didn't work.
In a previous installation, this error did not occur.

thanks,
Carlos.

**Comment by jmklix** [comment]
Are you making to install this into a clean folder? Are you also trying to install the aws-sdk-cpp? Which cmake version are you using?
I followed your commands and it builds without the same errors that you're seeing.

**Comment by bretambrose** [comment, marked as answer]
You appear to be running `cmake .` on the sample, which won't work.  You need to use `-DCMAKE_PREFIX_PATH=...` to tell cmake where to find the SDK.
